packages/@ionic/cli/src/commands/package/build.ts

```diff
@@ -22,6 +22,9 @@ const ANDROID_BUILD_TYPES = ['debug', 'release'];
 const IOS_BUILD_TYPES = ['development', 'ad-hoc', 'app-store', 'enterprise'];
 const APP_STORE_COMPATIBLE_TYPES = ['release', 'app-store', 'enterprise'];
 const BUILD_TYPES = ANDROID_BUILD_TYPES.concat(IOS_BUILD_TYPES);
+const ANDROID_ARTIFACT_TYPES = ['AAB', 'APK'];
```

Comment:
API endpoint only supports uppercase versions of these, should I just `.toUpperCase()` whatever user input. Should I allow case insensitiveness in api?

Reply:
Yeah, let's just `.toUpperCase()` the input.